scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -964,7 +975,7 @@ function Install-Packages(
                 Write-HostRed ('IoT Edge is already installed. To update, run "Update-IoTEdge". ' +
                     'Alternatively, if you want to finalize the installation, run "Initialize-IoTEdge".')
             }
-            throw
+            throw 'IoT Edge is already installed.'
```

Comment:
why extract the message on line 975 and use here?

Reply:
This message intends to summarize the if-else branches into one concise message:
 * "IoT Edge or the IoT Edge Moby Engine is installed in an invalid location."
 * "IoT Edge is already installed."

I am open to suggestions or leave the throw command with no message.